gui/MainWindow.cxx

```diff
@@ -1351,9 +1351,9 @@ void MainWindowPrivate::setActiveTool(AbstractTool* tool)
 
   auto const enableTools = !tool;
   auto const enableCancel = tool && tool->isCancelable();
-  foreach (auto const& tool, this->tools)
+  foreach (auto const& t, this->tools)
```

Comment:
I don't think replacing `tool` with `t` is an improvement to readability.  I would drop this change.

Reply:
name tool shadows the function parameter, which causes a warning, hence the change, not a big one, we can drop it here to address later

Reply:
ah, I see.  I missed that.  In that case it's fine to have here (or could just as easily be a separate PR,  we are due for a PR that just fixes a bunch of warnings)